subsys/net/lib/websocket/Kconfig

```diff
@@ -4,7 +4,7 @@
 #
 
 menuconfig WEBSOCKET
-	bool "Websocket support [EXPERIMENTAL]"
+	bool "Websocket support [EXPERIMENTAL, Deprecated]"
```

Comment:
hmmm, we are deprecating this with what? are we deprecating the feature itself or the API? This is confusing.

Reply:
There is nothing confusing here. The feature is provided by the API which is deprecated, so the these three features HTTP, websocket and SNTP will disappear too.

Reply:
Let me ask this differently, you are deprecating this in favour of what?

Reply:
I sent a mail to zephyr ml in the beginning of December asking if there are any users or willing maintainer volunteers for these APIs. None has been found so far. Atm there is no replacement APIs (using BSD sockets) proposed for these ones.

Reply:
> I sent a mail to zephyr ml in the beginning of December asking if there are any users or willing maintainer volunteers for these APIs.

I responded to both mails. In short: a) There's no need to haste with dirty-hacking replacements just for the purpose of replacement. It should be actual improvement (which takes time, given that we have more lower-level issues in queue still). b) It's ok to deprecate things, but not ok to remove, until replacements are available.

Reply:
while I agree in principe, this can't be open ended. The call for replacement was made very long time ago (as far as June last year) so the gap is not new and without someone stepping up and owning this and having a plan, I do not think we should just keep things in the tree indefinitely.